containers/hugging-face-inference/main.py

```diff
@@ -0,0 +1,46 @@
+import os
+
+from fastapi import FastAPI
+from llama_cpp import Llama
+from pydantic import BaseModel
+
+
+class Message(BaseModel):
+    content: str
+
+
+MODEL_FILE_NAME = os.environ["MODEL_FILE_NAME"]
+
+app = FastAPI()
+
+print("loading model starts", flush=True)
```

Comment:
nit: An alternative to adding `flush=True` to every print statement is to pass "PYTHONUNBUFFERED=1` as an env var (or even set it in the Dockerfile) to avoid buffering stdout